ONSdigital/sixteens
# Fix - Remove JS equal height

### What
Find and remove all instances of `js-equal-height` which sets fixed heights causing accessible word spacing extensions/bookmarklets to fail and overflow content.

### How to review
1. Visit a product page with or compedium landing page
1. See that running the bookmarklet causes text to overflow the content areas
1. Switch to this branch and babbage branch `fix/remove-auto-height`

### Who can review
Anyone but me
